index.html

```diff
@@ -2,30 +2,40 @@
 <html lang="en">
 
 <head>
-  <meta charset="UTF-8" />
-  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <link rel="stylesheet" href="index.css" />
+  <link rel="stylesheet" href="index.css"/>
   <title>Portfolio</title>
 </head>
-
 <body>
-    <nav class="navbar">
+  <header class="header">
+    <div class="container">
       <a class="logo" href="#">My Logo</a>
-      <button type="button" class="hamburger-button">&#8801;</button>
-      <div class="navbar-links">
-        <ul>
-          <li>Portfolio</li>
-          <li>About</li>
-          <li>Contact</li>
-         <li><button type="button" class="mail">&#9993;</button></li> 
+      <nav class="navbar">
+        <div class="hamburger-menu">
+          <div class="line line-1"></div>
+          <div class="line line-2"></div>
+          <div class="line line-3"></div>
+        </div>
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a href="#section1-nav" class="nav-link">Portfolio</a><hr>
+          </li>
+          <li class="nav-item">
+            <a href="#section2-nav" class="nav-link">About</a><hr>
+          </li>
+          <li class="nav-item">
+            <a href="#contact-section_nav" class="nav-link">Contact</a><hr>
+          </li>
```

Comment:
- [ ] As the previous reviewer requested, please fix mobile menu links issues. For example, when you click on the `about` link `My recent works` section appears. Kindly fix it 🙏